build.gradle

```diff
@@ -123,29 +128,32 @@ subprojects {
 
         azureSdkVersion = "1.2.21"
 
-        testcontainersVersion = "1.19.4"
+        testcontainersVersion = "1.19.7"
 
         testcontainersFakeGcsServerVersion = "0.2.0"
     }
 
     dependencies {
         compileOnly "org.apache.kafka:kafka-clients:$kafkaVersion"
-        compileOnly "org.apache.kafka:kafka-storage-api:$kafkaVersion"
+        compileOnly("org.apache.kafka:kafka-storage-api:$kafkaVersion")
```

Comment:
wondering if this is a style change only or it has some implications in the context of this PR?